### DISCUSSION TITLE
Please introduce yourself

### DISCUSSION THREAD

**Discussion by JohannesHoppe** [angular-schule/ngx-deploy-starter#3] [Show and tell]
Please write in this thread if you would use the starter and for which target. So that we know better what you need.

Based on your feedback, we could
* build a generator to make a start even easier in the future
* link to the website of your project, as soon as it is finished

**Comment by dianjuar** [comment]
Here's the first version of an Angular deployer for NPM 🙌 ngx-deploy-npm 🙌

https://www.npmjs.com/package/ngx-deploy-npm

## What it does
Deploy your Angular Package to NPM directly from the Angular CLI! 🚀

**Comment by JohannesHoppe** [comment]
Welcome to the club! 😊
